### ISSUE TITLE
Can you share the checkpoint of the finetune models?

### ISSUE THREAD

**Issue by wonzin** [TXH-mercury/VAST#25]
I have finetuned the VAST for QA tasks, but I’m not achieving the level of accuracy suggested in your paper. Could you possibly share the checkpoint of the fine-tuned models, particularly for the QA tasks? I would appreciate your assistance with this.
